023-AdvancedKubernetes/Coach/Solutions/07-data-volumes.md

```diff
@@ -5,11 +5,17 @@
 If availability zones are enabled, the disk will need to live in the same zone as the node that the pod is running on to attach.
 
 ```bash
+NRG=$(az aks show -n $AKS_NAME -g $RG -o json | jq '.nodeResourceGroup' -r)
+echo $NRG
+
 # if availability zones are NOT enabled
-az disk create -g MC_<resource group>_<cluster name>_<region> -n managed-disk-1 --size-gb 5
+az disk create -g $NRG -n managed-disk-1 --size-gb 5
 
 # if availability zones are enabled
-az disk create -g MC_<resource group>_<cluster name>_<region> -n managed-disk-1 --size-gb 5 --zone 1
+az disk create -g $NRG -n managed-disk-1 --size-gb 5 --zone 1
```

Comment:
Wouldn't this need to be the zone the pod is running in?
"--zone <zonePodIsRunningIn>"

The way you have it here reads like you're going to pin the disk to Zone 1?

Reply:
You're right, it would pin the disk to Zone 1.  Since the pod doesn't exist yet, when K8S tries to create it, it'll also pin the pod to Zone 1 too.  You must specify a value (e.g. 1,2,3) when using `--zone`, so it would have to get tied to somewhere.

Reply:
Ah...  so for a static volume, K8S will schedule the pod correctly where the disk is?  I didn't know that.  I thought it was the other way around.  For Dynamic, I know the disk gets created by K8S in the zone where the pod is scheduled.